crates/swc/tests/fixture/config/regenerator/import-path/1/output/index.js

```diff
@@ -1,4 +1,4 @@
-import _ts_generator from "@swc/helpers/src/_ts_generator.mjs";
+import { _ts_generator } from "@swc/helpers/_/_ts_generator";
```

Comment:
I think this will regress the bundle size, too.

```js
import { _ts_generator as e } from "@swc/helpers/_/_ts_generator";

e()
```

This is how it's optimized, but this looks problematic

Reply:
The test cases will be updated.
It will be changed to the following:

```JavaScript 
import { _ as _ts_generator } from "@swc/helpers/_/_ts_generator";
```

and with optimized phase:

```JavaScript 
import { _ as e } from "@swc/helpers/_/_ts_generator";

e()
```

Reply:
I see, thank you so much!